build.gradle

```diff
@@ -27,7 +27,8 @@ dependencies {
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-    version '2020.1.1'
+    version '2020.3'
     updateSinceUntilBuild false
     type 'CL'
+    setPlugins('com.intellij.clion', 'com.intellij.cidr.lang')
```

Comment:
My understanding is that it will be enough to  set it either here OR in the plugin.xml

Reply:
In 2020.3 they moved some CLion modules from the platform to new bundled plugins. This made it so that you had to first declare the plugins in build.gradle before in plugin.xml, otherwise the modules in plugin.xml wouldn't be found, and the project wouldn't build.

Reply:
I read their doc and it is confusing and it seem to imply that both are needed indeed.
But that doesn't really makes sense in my eyes because normally changes in build.gradle just overwrite the configuration in plugin.xml

I'll experiment on it